### PR TITLE
Feat/i18n crowdin

### PR DESCRIPTION
**Pull request by mihaic195**
Adds support for the following languages:
- Dutch
- French
- German
- Italian
- Portuguese
- Romanian
- Russian
- Spanish

Already extracted a lot of strings, but there are still more to go (including emails).

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]
excited for this

**Review by zomars — COMMENTED** [review]
Looking good! Left some comments there.

File: pages/[user].tsx [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-export default function User(props: inferSSRProps<typeof getStaticProps>) {
-  const { username } = props;
-  // data of query below will be will be prepopulated b/c of `getStaticProps`
-  const query = trpc.useQuery(["booking.userEventTypes", { username }]);
-  const { isReady } = useTheme(query.data?.user.theme);
-  if (!query.data) {
-    // this shold never happen as we do `blocking: true`
-    return <>...</>;
-  }
-  const { user, eventTypes } = query.data;
```
[the remark this answers is not in the document]
Comment (KATT) [reply pages/[user].tsx:26]: Yep - make sure to delete the actual function in `routers/viewer.ts` too!

[the remark this answers is not in the document]
Comment (zomars) [reply pages/[user].tsx:26]: Thanks for clarifying!

**Review by zomars — APPROVED** [review]

**Review by pumfleet — APPROVED** [review]

**Pull request merged**